README.md

```diff
@@ -38,16 +38,21 @@ Enjoy your new RPM.
 
 ### Ubuntu/Debian
 
-Building a .deb should be straight forward with:
+On recent Debian/Ubuntu distributions you should be able to install with:
+```
+# apt-get install TODO
```

Comment:
Can you please provide me with the name of the packages that users *should* install on Debian/Ubuntu? Thanks!